tokio-console/src/state/mod.rs

```diff
@@ -511,15 +511,20 @@ fn truncate_registry_path(s: String) -> String {
 
     static REGEX: OnceCell<Regex> = OnceCell::new();
     let regex = REGEX.get_or_init(|| {
-        Regex::new(r".*/\.cargo(/registry/src/[^/]*/|/git/checkouts/)")
-            .expect("failed to compile regex")
+        Regex::new(
+            r".*(/|\\)\.cargo(/|\\)(registry(/|\\)src(/|\\)[^/\\]*(/|\\)|git(/|\\)checkouts(/|\\))",
```

Comment:
It might make more sense to use character sets `[/\\]` instead of groups `(/|\\)` for the path separators. When I see groups I expect that we're extracting the contents.

That would make it like this:

```suggestion
            r".*[/\\]\.cargo[/\\](registry[/\\]src[/\\][^/\\]*[/\\]|git[/\\]checkouts[/\\])",
```

I think this regex is also getting **almost** too complex. I'd leave it for now, but if it becomes more so, I'd consider replacing it with either multiple regex or splitting the path into parts and then checking that way.

Reply:
Good suggestion! Updated!